jnigen/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.7.0
```

Comment:
I have not published 0.6.0 yet, so this can be in `0.6.0-dev.1` also in pubspec.yaml.

The way I do these change-logs is I start by writing `0.x.0-dev.0` and if it exists I add the next dev version `0.x.0-dev.1`, add the bullet points, and so on... (I also change the version in pubspec) The moment I want to publish I merge all of the `-dev.y`s, remove the `-dev.y` suffix from pubspec and change-logs and I publish. Of course the next breaking change now becomes `0.x+1.0-dev.0` again. 

Perhaps once we setup automatic publishing we can make this process better.